script.js

```diff
@@ -0,0 +1,57 @@
+console.log("Deliverables 1: " + products.items.length);
```

Comment:
Great work, some comments would make our code more accessible

Reply:
Thanks Scott. Have added comments to make it more readable.